src/reducers/notification.js

```diff
@@ -0,0 +1,18 @@
+import {
+    NOTIFICATION_SUCCESS,
+    NOTIFICATION_ERROR
+} from '../actionTypes/notification'
+import initialState from "../store/initialState"
+
+export default function reducer(state = initialState.notification, action) {
```

Comment:
There should be a state to store error. So,`NOTIFICATION_ERROR` should update error